projects/stingray/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: Stingray Investigation
+header:
+    image_fullwidth: "alpr.jpg"
+    permalink: /projects/stingray/
+---
+
+-----------------------
+
+The Lucy Parsons Labs has been central to investigating the use of IMSI Catchers (commonly known as Stingrays) by the Chicago Police Department for several years. Back in June of 2014, we successfully [sued for records of Stingray puchrases](http://arstechnica.com/tech-policy/2014/06/is-chicago-using-cell-tracking-devices-one-man-tries-to-find-out/). This allowed us to provide definitive proof, for the first time, that the Department had purchased these devices. The Stingrays were purchased using over four hundred thousand dollars from [civil asset forfeiture](http://www.chicagoreader.com/chicago/police-department-civil-forfeiture-investigation/Content?oid=23728922), a secret budget that CPD uses to buy high tech surveillance equipment. We also showed that information about Stingrays has been kept secret, because CPD signed a [non-disclosure agreement](https://www.scribd.com/document/272326979/CPD-Hailstorm-NDA) with the FBI and Harris Corporation,  as they have [done](https://www.cehrp.org/non-disclosure-agreements-between-fbi-and-local-law-enforcement/) [across](https://www.techdirt.com/articles/20151006/07585432448/tacoma-police-sued-over-heavily-redacted-stingray-non-disclosure-agreement.shtml) [the](https://www.wired.com/2014/03/harris-stingray-nda/) [country](http://arstechnica.com/tech-policy/2015/05/fbi-now-claims-its-stingray-nda-means-the-opposite-of-what-it-says/).
+
+Besides exposing these programs and forcing the spending [of more than several hundred thousand dollars](https://www.rt.com/usa/247597-chicago-police-stingray-costs/) by the City of Chicago fighting against transparency, the LPL has also been influential in other ways. We have presented at both [THOTCON](https://github.com/freddymartinez9/securitytalks/tree/master/Thotcon/) with a basic introduction on IMSI catchers and a surveillace self-defense talk at [DEFCON](https://www.youtube.com/embed/JyTb5mJOYLo). We signed on to the ACLU's [warrant requirement](https://lucyparsonslabs.com/stingray/stingraybill/) bill as a harm-reduction strategy as part of the war on drugs. Our litigation showed that CPD has been acquiring Trap and Trace [orders - pg.11](https://www.scribd.com/doc/295163479/CPD-Denied-motion-to-dismiss), which have a lower standard of proof than a warrant to authorize the invasive cell site simulator technology. This was particularly important as in other jurisdictions [such as Florida](http://arstechnica.com/tech-policy/2014/06/illinois-spent-over-250000-on-covert-cellular-tracking-equipment/), these devices have been used and law enforcement hides the source of information from defense attorneys. We also spend time teaching countersurveillance techniques to activists in Chicago, equipping them with rooted MotoE's and [SnoopSnitch](https://lucyparsonslabs.com/stingray/IMSI-Catching-Catching/)
```

Comment:
harm-reduction -> harm reduction

", these devices have been used and law enforcement hides the source of information from defense attorneys." -> "where these devices have been used, law enforcement has hidden the source of information from defense attorneys."

`[SnoopSnitch](https://lucyparsonslabs.com/stingray/IMSI-Catching-Catching/)` -> `[SnoopSnitch](https://lucyparsonslabs.com/stingray/IMSI-Catching-Catching/).`